readme.md

```diff
@@ -48,15 +46,15 @@ The source object to filter properties from.
 
 #### filter
 
-Type: `(sourceKey, sourceValue, source) => boolean`
+Type: `(sourceKey: string | symbol, sourceValue: any, source: object) => boolean`
```

Comment:
`Reflect.ownKeys()` returns indices as strings, so `sourceKey` cannot be a `number`.